core/templates/services/math-interactions.service.spec.ts

```diff
@@ -105,6 +105,18 @@ describe('MathInteractionsService', () => {
       'You have entered an invalid character: c. Please use only the ' +
       'characters x,y,z in your answer.');
 
+    expect(mathInteractionsService.validateAlgebraicExpression(
+      'ae^2 + 4b', ['a', 'b'])).toBeFalse();
+    expect(mathInteractionsService.getWarningText()).toBe(
+      'You have entered an invalid character: e. Please use only the ' +
+      'characters a,b in your answer.');
```

Comment:
This should be "variables" and not "characters" right? Because even ^, 2, + etc are characters right? (Technically speaking)..... Though unsure from a user standpoint, which would be better?

Reply:
Yup, that makes sense. I've changed it now.